examples/convention-cloud/run.sh

```diff
@@ -12,4 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-python3 -m sampletester.cli ${FLAGS:---xunit $(mktemp --suffix=.xml --tmpdir sampletester.xunit.XXXXX) -v detailed} --convention=cloud examples/convention-cloud/language.test.yaml examples/mock-samples/googleapis 
+sample-tester -v detailed --convention=cloud examples/convention-cloud/language.test.yaml examples/mock-samples/googleapis
```

Comment:
In the absence of unit tests for xUnit (for now, haven't gotten to this) let's keep `--xunit` to exercise the code. It has caught errors before. We can specify both `--xunit` and `-v`. If there's a portable way to specify the suffix, let's do that; otherwise, it doesn't matter if the file has the wrong suffix.

(We could potentially have `--xunit=/dev/null`, but I really want to avoid that because I want the xunit files to be accessible if we need to see them during development.)

Reply:
Soooo... instead, I added a test for xunit output.

It's a shell script, not pyunit, because that was faster/easier for me.  
You can do with it what you will.

Reply:
Thanks for adding the test. I expect I'll want to change it to pyunit at some point, but it's good for now.

I'd still prefer to have `--xunit`, though it's less pressing with this test (thanks again!). What's your reason for not having it?